Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -5855,6 +5856,78 @@ def com_google_fonts_check_metadata_can_render_samples(ttFont, family_metadata):
        yield PASS, "OK."
 
 
+def count_islands(imagefont, character):
+    from PIL import Image, ImageDraw
+    width, height = imagefont.getsize(character)
+    img = Image.new("RGBA", (width, height))
+    draw = ImageDraw.Draw(img)
+    draw.text((-2, 0), str(character), font=imagefont, fill="#000000")
+
+    visited = [False] * (width * height)
+    threshold = 150
```

Comment:
@m4rc1e, the reason why I am setting a threshold here is because the rendered glyph has antialiasing and it is not clear to me exactly which threshold would be ideal to distinguish to non-intersecting contours that are close enough to each other so that there's no full white pixel between them.

Maybe that's a flawed approach. Or there's some bug in my inplementation in this PR. Or it is just a matter of choosing a smart value for the threshold.

What do you think?